cypress/support/commands.js

```diff
@@ -22,4 +26,45 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const workerDate = {
+  firstName: faker.person.firstName(),
+  lastName: faker.person.lastName(),
+  email: faker.internet.email(),
+  age: Math.floor(Math.random() * (65 - 18 + 1)) + 18,
+  salary: Math.floor(Math.random() * (10000 - 100 + 1)) + 100,
+  department: faker.commerce.department()
+};
+
+Cypress.Commands.add('createWorker', () => {
```

Comment:
The 'createWorker' command does not have any kind of error handling or assertions to ensure the worker was created successfully.